### PR TITLE
Add check for layer0-api-endpoint

### PR DESCRIPTION
**Pull request by nehayward**
**What does this pull request do?**
A simple fix to flow.sh to ensure the api endpoint is set for layer0

### CODE REVIEW

File: scripts/flow.sh [hunk 1]
```diff
@@ -124,6 +124,11 @@ if [ -z $LAYER0_PREFIX ]; then
     exit 1
 fi
 
+if [ -z $LAYER0_API_ENDPOINT ]; then
```
Comment (zpatrick) [inline scripts/flow.sh]: where is this used?

Comment (nehayward) [reply scripts/flow.sh]: It needed it to do the delete flow. Or is flow only for local use? In which case I just needed the api running locally.

Comment (zpatrick) [reply scripts/flow.sh]: It can be used for either

Comment (nehayward) [reply scripts/flow.sh]: It's your call then to add this or not.

Comment (zpatrick) [reply scripts/flow.sh]: I'm going to not add this because I think it's more useful to have both options available.

**Pull request closed without merging**